<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>solution_added</class>
 <widget class="QWidget" name="solution_added">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>solution_added</string>
  </property>
  <widget class="QPushButton" name="close_window">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_add_solution">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>451</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>输入解决方法</string>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>391</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="add_manual">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string> </string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
